components/raftstore-v2/src/operation/query/mod.rs

```diff
@@ -141,13 +141,9 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             ));
         }
 
-        // TODO: add flashback_state check
```

Comment:
not added

Reply:
It's a clean up, flashback is supported in v2.